internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -145,23 +145,25 @@ func resourceKustoCluster() *pluginsdk.Resource {
 			"virtual_network_configuration": {
```

Comment:
Does this block need to be computed optional?

Reply:
I thought Hashicorp doesn't allow any property marked as computed optional and user has to manually ignore changes for Computed Optional property. Should we added this to the block?